_source/_posts/2019-02-15-user-migration-the-definitive-guide.md

```diff
@@ -307,7 +307,7 @@ And, if you're looking for a user management service that fits that criteria, yo
 
 ## User Migration Example
 
-If you'd like to see an example of a user migration tool, you can check out this [.NET migration program](https://github.com/oktadeveloper/okta-user-migration-dotnet) we built.
+If you'd like to see an example of a user migration tool, you can check out this [.NET migration program](https://github.com/andyclarke-okta/cvs_liveMigration) we built.
```

Comment:
I'm not sure this is a great idea. Our repo has a README with instructions. The new repo has nothing.

Reply:
Any thoughts @rdegges?